test/tools/PSxUnit/XunitProjectAssembly2.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.IO;
+
+namespace Xunit.Runner.DotNet
+{
+    class XunitProjectAssembly2 : XunitProjectAssembly
+    {
+        public TestAssemblyConfiguration ConfigurationStream
+            => LoadConfiguration(AssemblyFilename);
+
+        static TestAssemblyConfiguration LoadConfiguration(string assemblyName)
+        {
+            var stream = GetConfigurationStreamForAssembly(assemblyName);
+            return stream == null ? new TestAssemblyConfiguration() : ConfigReader.Load(stream);
+        }
+
+        static Stream GetConfigurationStreamForAssembly(string assemblyFilename)
+        {
+            // get parent directory
+            var directory = Path.GetDirectoryName(assemblyFilename);
+            var assemblyName = Path.GetFileNameWithoutExtension(assemblyFilename);
+
+            var pathWithAssemblyName = Path.Combine(directory, assemblyName);
+
+            // See if there's a directory with the assm name. this might be the case for appx
+            if (Directory.Exists(pathWithAssemblyName))
+            {
+                if (File.Exists(Path.Combine(pathWithAssemblyName, $"{assemblyName}.xunit.runner.json")))
+                    return File.OpenRead(Path.Combine(pathWithAssemblyName, $"{assemblyName}.xunit.runner.json"));
```

Comment:
The same Path.Combine() is done twice. It should be something like:

var runnerJson = Path.Combine(pathWithAssemblyName, $"{assemblyName}.xunit.runner.json")

if(File.Exists(runnerJson)
{
   return File.OpenRead(runnerJson);
}